lnbits/extensions/cashu/models.py

```diff
@@ -68,10 +68,11 @@ def from_row(cls, row: Row):
     def from_dict(cls, d: dict):
         assert "secret" in d, "no secret in proof"
         assert "amount" in d, "no amount in proof"
+        assert "C" in d, "no C in proof"
         return cls(
-            amount=d.get("amount"),
-            C=d.get("C"),
-            secret=d.get("secret"),
+            amount=d["amount"],
```

Comment:
Q: why is `[]` preferred to `.get("")`

Reply:
get always returns an Optional and because they are required and asserted above its fine to directly access them.